### DISCUSSION TITLE
Option to disable Contribution Graph.

### DISCUSSION THREAD

**Discussion by shenlebantongying** [community/community#4119] [Other Feature Feedback, Questions, & Ideas]
A lot of people don't like and even hate it, they might:

+ Don't want to others know their "spam" contributions like fixing typos, translations, and mini fixes <https://github.com/isaacs/github/issues/142>
+ Don't want to annoy their employer  <https://github.com/isaacs/github/issues/627>
+ Want more privacy <https://github.com/isaacs/github/issues/1840>
+ Dislike gamification <https://github.com/isaacs/github/issues/1234>
+ Incorrect representation for open source contributions <https://github.com/isaacs/github/issues/1290>
+ etc.

There are more discussions floating around the internet.

**Comment by Levi-Lesches** [comment]
These issues had tons of support over at the old repo -- 1,750 upvotes between the linked issues. GitHub, please take that into consideration here.

**Comment by Levi-Lesches** [comment]
Should we close this as a duplicate of #4098?
